CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog 1.0.0].
 
 ## [Unreleased]
 
+## [Release 14.0.0]
+
```

Comment:
need to add


`[Add new Document.delete() method](https://github.com/nationalarchives/ds-caselaw-custom-api-client/commit/527ad8d2aced1d09f95a03f91fb2f89715a939c5)`

and 

- Generalised the set judgment metadata methods to set document metadata methods
specifically for name, court and date